manifests/repo.pp

```diff
@@ -119,6 +119,16 @@
         Apt::Source['zabbix'] -> Package<|tag == 'zabbix'|>
         Class['Apt::Update']  -> Package<|tag == 'zabbix'|>
       }
+      'Suse': {
+        $minorrelease = $facts['os']['release']['minor']
+        zypprepo { 'monitoring':
```

Comment:
This is a new 'soft' dependency.  So even though it might not or perhaps shouldn't go in the metadata.json could you update the README?